egui_glium/src/painter.rs

```diff
@@ -319,3 +319,22 @@ impl Painter {
         }
     }
 }
+
+impl epi::NativeTexture for Painter {
+    type Texture = glium::texture::srgb_texture2d::SrgbTexture2d;
+
+    fn bind_native_texture(&mut self, native: Self::Texture) -> egui::TextureId {
+        self.register_glium_texture(native)
```

Comment:
I think `register_native_texture` is a better name, and just unify the functions

Reply:
i changed it